modules/advenueBidAdapter.js

```diff
@@ -0,0 +1,115 @@
+import { registerBidder } from 'src/adapters/bidderFactory';
+import { BANNER, NATIVE, VIDEO } from 'src/mediaTypes';
+import * as utils from 'src/utils';
+
+const BIDDER_CODE = 'advenue';
+const URL = '//supply.advenuemedia.co.uk/?c=o&m=multi';
+const URL_SYNC = '//supply.advenuemedia.co.uk/?c=o&m=cookie';
+
+function isBidResponseValid(bid) {
```

Comment:
Prebid core is already doing all these checks. No need to add validate your response